deployment/2_deploy_contracts.js

```diff
@@ -59,7 +59,7 @@ async function main() {
 
   // Deploy Module Registry
   const ModuleRegistryWrapper = await ModuleRegistry.new();
-  const AuthoriserWrapper = await Authoriser.new();
+  const AuthoriserWrapper = await Authoriser.new(newConfig.settings.securityPeriod);
```

Comment:
We should add a new `timelockPeriod`  entry in the config and that's what the constructor should use.